api/external/compliance/profiles/profiles.proto

```diff
@@ -142,6 +142,28 @@ service ProfilesService {
 			action: "compliance:profiles:list"
 		};
 	};
+	rpc MetaSearch(Sha256) returns (Missing) {
```

Comment:
let's make a note to come back and add documentation comments for this

Reply:
I'll add it to my next PR for ingestion. Thank you!

Reply:
Victoria, I pushed in this (https://github.com/chef/automate/commit/efaf0fb4d4dea91877db0b77253952f0df36fe60) commit of https://github.com/chef/automate/pull/3302 the docs for `/metasearch`